cdap-tms/src/main/java/co/cask/cdap/messaging/store/hbase/HBaseTableFactory.java

```diff
@@ -53,6 +53,7 @@
 import org.apache.hadoop.hbase.client.HBaseAdmin;
 import org.apache.hadoop.hbase.client.HTable;
 import org.apache.hadoop.hbase.client.RetriesExhaustedWithDetailsException;
+import org.apache.hadoop.hbase.regionserver.DisabledRegionSplitPolicy;
```

Comment:
Is this available in all supported HBase versions? Would we know if not, because some ITNs will fail? Or would HBase simply log a warning?